### DISCUSSION TITLE
Question: MultiDataEntry with given list items as "options" (not from db tables)?

### DISCUSSION THREAD

**Discussion by traveller195** [tomalrussell/colouring-london#1105] [Q&A]
Hello,
I have a question: @matkoniecz 
I want to use MultiDataEntry with a given selection of strings, which can be chosen. Not only one row (it would be SelectDataEntry), but multiple rows. And also not free text field. So, the given selection should be shown as dropdown selection.

Currently, MultiDataEntry is used for landuse and - if I understand it correctly - those categories will be loaded from db tables "reference tables" directly from the database.

What I need now is just an parameter like "`options`" directly for MultiEntryData, so that I can load the given selection e.g. from data-fields-config.ts from `"items: ["string1", "string2", "string3"]"` and assign it as a list for "options" parameter.

Similar to SelectDataEntry, which offers the nice feature "`options`".

Is there already existing a similar parameter for MultiDataEntry?
../app/src/frontend/building/data-components/multi-data-entry/multi-data-entry.tsx

thanks in advance :-)
best

**Comment by matkoniecz** [comment]
> What I need now is just an parameter like "options" directly for MultiEntryData, so that I can load the given selection

You can look at "Is the building a home/domestic building?" which is not loading options from database (though it is not MultiEntryData)

You can try grepping for all uses of MultiDataEntry class, but I do not remember any exact match.

**Reply by traveller195** [reply]
alright, thanks

**Reply by traveller195** [reply]
yes, you are right. as you mentioned "is_domestic" is SelectDataEntry, not MultiDataEntry

but I just realized, that there is a Multi-Select-Data-Entry
../app/src/frontend/building/data-components/multi-select-data-entry.tsx

maybe this is the solution for me. At least, it contains "options" parameter. But I am not sure whether it is already ready coded...I will test it
